packages/integrations/vue/src/editor.cts

```diff
@@ -40,20 +41,19 @@ export function toTSX(code: string, className: string): string {
 				// TODO. Find a way to support generics when using defineProps without passing explicit types.
 				// Right now something like this `defineProps({ prop: { type: Array as PropType<T[]> } })`
 				//  won't be correctly typed in Astro.
-				const defineProps = scriptSetup.content.match(/defineProps\([\s\S]+\)/);
-
+				const defineProps = codeWithoutComments.match(/defineProps\([\s\S]+?\)/);
 				if (defineProps) {
 					result = `
-					import { defineProps } from '@vue/runtime-core';
+						import { defineProps } from 'vue';
```

Comment:
Is there a reason `@vue/runtime-core` is changed to `vue`. We should also remove the indent changes

Reply:
In my case with pnpm, I found that `@vue/runtime-core` might not be installed, In order to ensure the correct import of defineProps, I changed it to `vue` because it will be installed.